tests/test_builders/test_build_html.py

```diff
@@ -1,15 +1,24 @@
 """Test the HTML builder and check output against XPath."""
 
+from __future__ import annotations
+
+import os
 import posixpath
 import re
+from typing import TYPE_CHECKING
 
 import pytest
 
 from sphinx.builders.html import validate_html_extra_path, validate_html_static_path
 from sphinx.deprecation import RemovedInSphinx80Warning
 from sphinx.errors import ConfigError
+from sphinx.util.console import strip_colors
 from sphinx.util.inventory import InventoryFile
 
+if TYPE_CHECKING:
+
+    from sphinx.application import Sphinx
+
```

Comment:
Are you planning to add use of this as a typehint in this branch?

Reply:
I used it in the local handler actually. But maybe it's a bit too much. Let's revert it for now.